examples/custom-resources/jwks/README.md

```diff
@@ -0,0 +1,162 @@
+# JWKS
+
+In this example we deploy a web application, configure load balancing with a VirtualServer, and apply a JWT policy.
+Instead of using a local secret to verify the client request such as in the [jwt](https://github.com/nginxinc/kubernetes-ingress/tree/main/examples/custom-resources/jwt) example, we will define an external Identity Provider (IdP) using the `JwksURI` field.
+
+We will be using a deployment of [KeyCloak](https://www.keycloak.org/) to work as our IdP in this example.
+
```

Comment:
Please add the following.

"In this example, we will be deploying Keycloak as a single container for example purposes and exposing it with Ingress controller."

We want customers understand to have a good grasp on this example setup (Keycloak and NIC)